breakwater-parser/src/implementations/simple.rs

```diff
@@ -184,6 +184,7 @@ impl Parser for SimpleParser {
     }
 }
 
+#[inline]
```

Comment:
I would have never expected the `const` function to be called at runtime :thinking: 
What do you think of even putting in some constants just to be 100% (ok, lets say 99% ^^) sure?
`const PX_PATTERN: u64 = string_to_number(b"PX \0\0\0\0\0");` and using that? (same applies for all other `string_to_number`) calls

Reply:
With this inline, the function is nowhere to be found. I am currently trying to read through the disassembly, and adding inline definitely fixed this issue.